python/Cargo.toml

```diff
@@ -25,9 +25,12 @@ geo-index = { path = "../", features = ["rayon"] }
 # This is the fork used by polars
 # https://github.com/pola-rs/polars/blob/fac700d9670feb57f1df32beaeee38377725fccf/py-polars/Cargo.toml#L33-L35
 numpy = { git = "https://github.com/stinodego/rust-numpy.git", rev = "9ba9962ae57ba26e35babdce6f179edf5fe5b9c8", default-features = false }
-pyo3 = { version = "0.21", features = ["abi3-py38"] }
+pyo3 = { version = "0.21.0", features = ["multiple-pymethods", "macros"] }
```

Comment:
I won't include `multiple-pymethods` as a feature because that prevents this package from compiling for pyodide. See https://github.com/PyO3/pyo3/issues/2517#issuecomment-1188629768

Reply:
Yeah, not actually necessary anyway, I flipped it on prematurely.